LoopKit/QuantityFormatter.swift

```diff
@@ -147,7 +147,9 @@ public extension HKUnit {
     }
 
     var maxFractionDigits: Int {
-        if self == HKUnit.gram().unitDivided(by: .internationalUnit()) {
+        if self == .internationalUnitsPerHour {
+            return 3
```

Comment:
[nit] How does specific ranges of rates get affected by this? I mean, say between 5-10 IU/hr the pump can only take 0.01 resolution.

Reply:
This property determines the upper bound on digits; 3 won't be used if only 2 are necessary.